packages/syft/src/syft/service/user/user.py

```diff
@@ -112,12 +112,23 @@ def migrate_server_user_update_v1_current() -> list[Callable]:
     return [
         make_set_default("reset_token", None),
         make_set_default("reset_token_date", None),
+        make_set_default(
```

Comment:
do you have to do the drop first like in the reverse migration?

Reply:
Fixed by: [470cae8](https://github.com/OpenMined/PySyft/pull/9052/commits/470cae83db0c59070c5ebbeef48f4d8d8a76a24c)